api/v1beta2/moduletemplate_types.go

```diff
@@ -80,6 +80,11 @@ type ModuleTemplateSpec struct {
 	// +kubebuilder:validation:MinLength:=3
 	Channel string `json:"channel"`
 
+	// Mandatory indicates whether the module is mandatory. It is used to enforce the installation of the module with
+	// its configuration in all runtime clusters.
+	// +optional
+	Mandatory bool `json:"mandatory"`
```

Comment:
This field should also introduce into `v1beta1`, `v1beta1` is current storage version, if you only introduce here, the value will be lost during conversion. @nesmabadr  @jeremyharisch